metacat-connector-druid/src/main/java/com/netflix/metacat/connector/druid/DruidConnectorFactory.java

```diff
@@ -0,0 +1,74 @@
+/*
+ *  Copyright 2017 Netflix, Inc.
+ *
+ *     Licensed under the Apache License, Version 2.0 (the "License");
+ *     you may not use this file except in compliance with the License.
+ *     You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *     Unless required by applicable law or agreed to in writing, software
+ *     distributed under the License is distributed on an "AS IS" BASIS,
+ *     WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *     See the License for the specific language governing permissions and
+ *     limitations under the License.
+ */
+
+package com.netflix.metacat.connector.druid;
+
+import com.netflix.metacat.common.server.connectors.ConnectorContext;
+import com.netflix.metacat.common.server.connectors.ConnectorDatabaseService;
+import com.netflix.metacat.common.server.connectors.ConnectorPartitionService;
+import com.netflix.metacat.common.server.connectors.ConnectorTableService;
+import com.netflix.metacat.common.server.connectors.SpringConnectorFactory;
+import com.netflix.metacat.connector.druid.configs.DruidConnectorConfig;
+import com.netflix.metacat.connector.druid.configs.DruidHttpClientConfig;
+import com.netflix.metacat.connector.druid.converter.DruidConnectorInfoConverter;
+
+/**
+ * Druid Connector Factory.
+ *
+ * @author zhenl
+ * @since 1.1.0
+ */
+public class DruidConnectorFactory extends SpringConnectorFactory {
+    /**
+     * Constructor.
+     *
+     * @param catalogName      catalog name.
+     * @param connectorContext connector config
+     */
+    DruidConnectorFactory(
+        final String catalogName,
+        final DruidConnectorInfoConverter druidConnectorInfoConverter,
+        final ConnectorContext connectorContext
+    ) {
+        super(catalogName, druidConnectorInfoConverter, connectorContext);
+        registerClazz(DruidConnectorConfig.class, DruidHttpClientConfig.class);
```

Comment:
You might want to make it explicit here that this is calling `super` or at least `this`... I wasn't really sure what this was doing.